doc/architecture/decisions/0002-pathogens-portal-2-0-architecture.md

```diff
@@ -0,0 +1,150 @@
+2. **Hard to implement data standards:** in the current workflow, there is no testing related to ensuring that the updated data and associated metadata comply with given standard, which provides very limited help to researchers.
+3. **Limited automation**: in the absence of a dedicated backend, data processing and visualisation updates require manual intervention and script management, and we cannot implement automated workflows to support the sustainable addition of new features.
+
+#### Need to redesign the Swedish Pathogens Portal
+
+This ADR proposes a change in architecture that will address the limitations of the Swedish Pathogens Portal by transitioning it from a static website to a dynamic web application.
+This will allow developers to concentrate their efforts on serving the infectious disease and pandemic preparedness research communities in Sweden more effectively by maintaining fewer services and adding new, data-centred features and automated workflows.
+
+## Decision
+
+We will transform the Swedish Pathogens Portal from a static website into a **service-oriented, modular web application**, eliminating all supporting services such as Blobserver and Form Manager. This new architecture comprises independently deployable components with clearly defined responsibilities, enabling the Pathogens Portal to expand in terms of both functionality and usage.
+
+![Updating data workflow in current Pathogens Portal](../../figures/0002-portal2.0-architecture.jpg)
+
+*Figure 3: Proposal for new Swedish Pathogens Portal architecture*
+
+### Core Technology Stack
+
+* **Frontend**: [Next.js](https://nextjs.org/). Responsible for rendering the user interface and interacting with APIs. Supports hybrid rendering (static and dynamic) and offers high performance for modern web delivery.
+
+* **Backend**: [Django](https://www.djangoproject.com/). Acts as the system’s core, exposing API endpoints via Django REST framework, handling authentication, business logic, and orchestrating integrations with external systems.
+
+* **Database**: [PostgreSQL](https://www.postgresql.org/). Stores structured content, metadata, configuration data, and supports advanced querying through features like JSONB and full-text search.
+
+* **External Services**: These services will be consumed by the backend via their APIs:
+
+  * **Portal storage solution:** for data uploads and file access
+  * **Integrated Data Services (IDS):** for harmonised metadata and research objects
+  * **External repositories:** e.g. publication indexes, genomic data archives
+
+### Why This Architecture Approach?
+
+We chose a **service-oriented architecture (SOA)** because it provides:
+
+* **Loose coupling:** Components can evolve independently, enabling isolated deployments and targeted updates without affecting other parts of the application.
```

Comment:
While true, please make sure that you have enough people power for development and mainetenance of all components involved:)

And take into account additional difficulty of devepoment of such a service, where you need to run two webservers in parallel in order for them to communicate. This difficulty could be mitigated by htmx though, while adding the dynamicity to the webpages